src/lib/discord/commands/_repositories.ts

```diff
@@ -1,23 +1,93 @@
-/**
- * @TODO replace with GitHub API call
- */
+import { getRepos } from '../../github/queries'
+
+const nameToAlias = new Map<string, string>([
+  ['amplify-cli', 'cli'],
+  ['amplify-ios', 'ios'],
+  ['discord-bot', 'discord-bot'],
+  ['amplify-ui', 'ui'],
+  ['amplify-js', 'js'],
+  ['docs', 'docs'],
+  ['amplify-android', 'android'],
+  ['amplify-flutter', 'flutter'],
+  ['amplify-category-api', 'category-api'],
+  ['amplify-cli-export-construct', 'cli-export-construct'],
+  ['amplify-codegen', 'codegen'],
+  ['amplify-adminui', 'adminui'],
+  ['amplify-hosting', 'hosting'],
+  ['Test-Repository', 'TEST'],
+])
+
+const alias = (name: string) => nameToAlias.get(name) ?? name
+
 async function fetchRepositories() {
-  const github = 'https://github.com'
-  const org = 'aws-amplify'
-  const createRepoUrl = (repo: string) => `${github}/${org}/${repo}`
-  return {
-    android: createRepoUrl('amplify-android'),
-    bot: createRepoUrl('discord-bot'),
-    cli: createRepoUrl('amplify-cli'),
-    docs: createRepoUrl('docs'),
-    flutter: createRepoUrl('amplify-flutter'),
-    hosting: createRepoUrl('amplify-hosting'),
-    ios: createRepoUrl('amplify-ios'),
-    js: createRepoUrl('amplify-js'),
-    ui: createRepoUrl('amplify-ui'),
+  try {
+    const organization = await getRepos()
+    if (organization?.repositories?.edges?.length) {
+      const repos = organization.repositories.edges
+      return repos
+      .filter((repo) => nameToAlias.get(repo.node.name))
+      .reduce((accumulator, repo) => {
+        return { ...accumulator, [alias(repo.node.name)]: repo.node.url }
+      }, {})
+    }
+  } catch (error) {
+    console.error(`Error fetching repositories: ${error.message}`)
+  }
+  return {}
+}
+
+const containsQA = (category: Category) =>
+  category?.name === process.env.GITHUB_DISCUSSION_CATEGORY
```

Comment:
is this a new env var? can you add this to `.env.sample` with a small note? Also I think this is a great prop to store in the `Configuration` model 👀